### PR TITLE
Feature/75 jenkins graph importer analysis core

### PR DESCRIPTION
**Pull request by cookersjs**
- JF for creating graph database and analysis core for release using shared groovy library

### CODE REVIEW

**Review by SolomonShorser-OICR — APPROVED** [review]

File: jenkins/Jenkinsfile-graph-importer-analysis-core [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+					dir("graph-importer"){
+						// Bulds jar file for graph-importer
+						utils.buildJarFile()
+						
+						// This generates the graph database.
+						withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+							sh "java -jar target/GraphImporter-jar-with-dependencies.jar --name ${env.REACTOME_DB} --user $user --password $pass --neo4j /tmp/graph.db --interactions"
+							sh "cp -r /tmp/graph.db/ ."
+							sh "sudo service tomcat7 stop"
+							sh "sudo service neo4j stop"
```
Comment (SolomonShorser-OICR) [inline jenkins/Jenkinsfile-graph-importer-analysis-core:51]: Maybe add a note about _why_ these services need to be stopped for something that doesn't even use them.

Comment (cookersjs) [reply jenkins/Jenkinsfile-graph-importer-analysis-core:52]: Done - thanks for pointing that out

**Review by SolomonShorser-OICR — APPROVED** [review]

**Review by jweiser — APPROVED** [review]

**Review by SolomonShorser-OICR — DISMISSED** [review]

**Comment by cookersjs** [comment]
@SolomonShorser-OICR @jweiser had to resolve merge conflict

**Review by SolomonShorser-OICR — APPROVED** [review]

**Comment by SolomonShorser-OICR** [comment]
Next time, please try to resolve the merge conflicts before requesting review.

**Comment by cookersjs** [comment]
> Next time, please try to resolve the merge conflicts before requesting review.

I didn't realize 'merges' in conflicts are actually merging the resolution into the current branch, not completing the merge into the destination branch, I actually waited until I'd received approval before resolving it haha... live and learn!

**Pull request merged**